### PR TITLE
fix(Grid): Row justify setting `space-evenly` does not work

### LINKED ISSUE #41675: Row justify space-evenly is broken
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/s/typesetting-antd-5-4-0-forked-o4wliu)

### Steps to reproduce

1. Create a `Row` with multiple `Col`'s
2. Set `justify` to `space-evenly`

### What is expected?

The columns are evenly spaced

### What is actually happening?

The columns are not spaced at all.

| Environment | Info |
| --- | --- |
| antd | 5.4.0 |
| React | 18.2.0 |
| System | Windows 11 Pro 10.0.22621 Build 22621 |
| Browser | Chrome 111.0.5563.147 |

---
This behaviour can also be seen on https://ant.design/components/grid.
It seems broken since v5, when setting the version to '4.x' on the website the spacing works.

v4
image

v5
image

### PR DESCRIPTION
**Pull request by Yuiai01**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/41675
### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix `Row` justify setting `space-evenly` does not work           |
| 🇨🇳 Chinese | 修复 `Row` 的 `justify` 设置为 `space-evenly` 无效        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 126f951</samp>

Added a new feature for the `Row` component to allow even spacing of columns using the `space-evenly` prop. Updated the style file and the test file to support the new feature.

### <samp>🤖 Generated by Copilot at 126f951</samp>

* Add `justify="space-evenly"` prop to `Row` component to allow equal spacing between columns ([link](https://github.com/ant-design/ant-design/pull/41679/files?diff=unified&w=0#diff-c8d24dba3dd3cae5f0db517022a284eb2fdc7f70f00007f29af301cb358e5432L49-R56))
* Update `genGridRowStyle` function to handle the new prop and generate the appropriate CSS class and property ([link](https://github.com/ant-design/ant-design/pull/41679/files?diff=unified&w=0#diff-c8d24dba3dd3cae5f0db517022a284eb2fdc7f70f00007f29af301cb358e5432L49-R56))
* Reorder imports in `index.test.tsx` to follow alphabetical order ([link](https://github.com/ant-design/ant-design/pull/41679/files?diff=unified&w=0#diff-03fce7d2dec4a5ee8fd50ed5ca554266305b0b5e9546cea5206f5154b2acca3dL6-R7))
* Add test case for `Row` component with `justify="space-evenly"` prop in `index.test.tsx` ([link](https://github.com/ant-design/ant-design/pull/41679/files?diff=unified&w=0#diff-03fce7d2dec4a5ee8fd50ed5ca554266305b0b5e9546cea5206f5154b2acca3dR235-R247))

### CODE REVIEW

File: components/grid/style/index.ts [hunk 1]
```diff
@@ -46,10 +46,14 @@ const genGridRowStyle: GenerateStyle<GridRowToken> = (token): CSSObject => {
...
       },
 
-      '&-space-around ': {
+      '&-space-around': {
         justifyContent: 'space-around',
       },
 
+      '&-space-evenly': {
+        justifyContent: 'space-evenly',
+      },
```
Comment (afc163) [inline components/grid/style/index.ts:55]: 是之前迁移迁丢了么？

Comment (yoyo837) [reply components/grid/style/index.ts:55]: 可以查一下commit，我下班了不然我就直接查一下了，手机不方便

Comment (Yuiai01) [reply components/grid/style/index.ts:55]: 看了一下 commit，三个月前创建的该文件，那个时候就没有
image

Comment (yoyo837) [reply components/grid/style/index.ts:55]: css in js从less文件搬过来的，看看原来的less文件？

Comment (Yuiai01) [reply components/grid/style/index.ts:55]: > css in js从less文件搬过来的，看看原来的less文件？

好 我去康康

Comment (Yuiai01) [reply components/grid/style/index.ts:55]: 之前是有的，看来是丢了
image

**Review by afc163 — APPROVED** [review]

**Pull request merged**